### DISCUSSION TITLE
why cell voltages in red

### DISCUSSION THREAD

**Discussion by Ericpeetoom** [Louisvdw/dbus-serialbattery#447] [Q&A] [answered]
Bon dia,
![red](https://user-images.githubusercontent.com/124325971/216497411-7ef6a586-39ba-49ee-b368-2a57af71f8a8.png)
I use a victron Quattro with 2 times an EG4 life power 4 batteries controlled by Raspberry venus 3.00 everything works okay but it shows the cell voltages in red why is this?.
thanks

**Comment by Louisvdw** [comment, marked as answer]
The red should turn while when your battery is closer to empty, correct? Or does it stay red always?

The red means that cell is being balanced. So normally you would see the red only appearing on the cells that are higher than the rest, and only if the balance threashold has been reach. Your cells does seem all balanced (same voltage level). If it keeps the red, then it could be that the flag for the EG4 is not correctly set.

**Reply by Ericpeetoom** [reply]
Hi,
No, he always indicates it in red, it doesn't change
How do I set the Flag for my EG4 correctly?.
Thanks
Eric from Curacao

**Reply by Alstivi** [reply]
I also have the same Problem with a Daly 16s BMS on Cerbo GX with UART to USB Adapter. 
Installed Driverversion 14.3.

![Bild_2023-05-24_192825349](https://github.com/Louisvdw/dbus-serialbattery/assets/111471322/f7330ff2-d44f-4d3d-bdbc-690e7492c31f)

Have you already found a solution for this problem @Ericpeetoom  @Louisvdw?
Thank you for your great work @Louisvdw

**Reply by mr-manuel** [reply]
Just install the latest beta from my fork: https://github.com/mr-manuel/venus-os_dbus-serialbattery/releases
